### PR TITLE
New post: What is Cloud Storage

### PR DESCRIPTION
**Pull request by cobusbernard**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
Recreated the PR, please continue the review here from old #331. Due to making the repo public.

**Comment by cobusbernard** [comment]
@gaonkarr / @damonk10 : Please make sure to look at the comments left on the old PR as they weren't copied over.

**Comment by gaonkarr** [comment]
Yes my major comments were for 1. the image - added to repo but not mentioned in the blog; 2. links are not added in the conclusion; 3. Couple of extra spaces highlighted by markdown plugin. Rest were minor editorial comments.

File: posts/what-is-cloud-storage/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Conclusion
+
+Cloud storage is the unsung hero of the modern era: humanity is creating some 100 billion terabytes of data each year at this point, and that number is rapidly rising. But numbers that big are hard to visualize. So think about it like this: if every byte of data we created last year was a millimeter, our data would reach the sun... and back... over 300 million times. Okay, that still doesn’t help. Basically, the amount of data here is unimaginably big.
+
+And this digital reality depends on an infrastructure most of us take for granted. If you want to learn more about cloud storage, check out my video on the topic, or watch my discussion with Curtis Evans where we dive deeper into modern cloud storage implementations.
```
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:74]: Please add the relevant links with tracking wherever applicable

File: posts/what-is-cloud-storage/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Takeaways
+
+Okay, we know the three kinds of storage — object, block, and file. But what’s the takeaway here for most people? Well, a few observations:
+
+First, regular people use storage in really different ways from big companies. Object storage is definitely the most common sort of storage you encounter everyday via the internet — because for the most part, we’re just retrieving media. But companies are using a lot of block storage (because it enables better analysis of data and it performs really well at scale) and file storage (because it allows us to collaborate without corrupting data).
+
+The other high-level takeaway:
+
+Storage and computing used to always be linked. You bought a laptop or desktop computer, and how much computing power it had generally corresponded with how much storage it would have. Now we’re now in a situation where compute and storage in a cloud environment can become decoupled. Not only do cloud storage and compute dwarf traditional local resources — you can access vast quantities of both using machines with highly variable individual capabilities.
```
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:68]: Now is repeated? "Now we are in situation..."

File: posts/what-is-cloud-storage/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  - storage
+  - block-storage
+  - object-storage
+  - file-storage
+authorGithubAlias: damonk10
+authorName: David Priest
+date: 2023-05-25
+---
+
+In the modern world, we all need storage, whether it’s the garage hiding your dusty exercise equipment, or your kitchen cupboards helping you find that one spice you really need, or that space under your stairs for stashing recently-orphaned child wizards. Well cloud storage has just as many options as the real world — and you use them a lot more often than you might realize.
```
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:16]: You know best, but should it be : "Well,"

File: posts/what-is-cloud-storage/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Object storage also keeps track of metadata. Put simply, metadata is data about data. Different files have to be transferred and displayed differently based on their type — and it’s useful to store ancillary information based on the types of files, too. For instance, you may want to know when and where a photo was taken, but for a written doc, you just need to know the author. Object storage knows how to keep track of this sort of metadata.
+
+In short, object storage keeps each file intact, and keeps track of its unique metadata, which is valuable when storing lots of different types of files.
+
+## Block Storage
+
+Block storage generally gets a lot more use in business settings. It tends to be more highly performant than object storage, because it’s often supporting huge databases getting constant use — and that require extremely low latency in order to carry out their complex tasks effectively.
```
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:36]: Can we break this down a bit? Are we saying it's highly performant and thats why it is used for databases or other way around?

File: posts/what-is-cloud-storage/index.md (hunk 2 above)
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:68]: should we add 'the' for better readability? "Not only do cloud storage and compute dwarf the traditional local resources..."

File: posts/what-is-cloud-storage/index.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+tags:
+  - basics
+  - cloud
+  - storage
+  - block-storage
+  - object-storage
+  - file-storage
+authorGithubAlias: damonk10
+authorName: David Priest
+date: 2023-05-25
```
Comment (gaonkarr) [inline posts/what-is-cloud-storage/index.md:13]: JPEG Image is added, but never referenced in the blog. Do you wish to add it to fontmatter?

**Pull request merged**